packages/fiber/src/core/index.tsx

```diff
@@ -286,7 +286,7 @@ function createRoot<TCanvas extends Element>(canvas: TCanvas): ReconcilerRoot<TC
       // Safely set color management if available.
       // Avoid accessing THREE.ColorManagement to play nice with older versions
       if ('ColorManagement' in THREE) {
-        setDeep(THREE, legacy, ['ColorManagement', 'legacyMode'])
+        setDeep(THREE, !legacy, ['ColorManagement', 'enabled'])
```

Comment:
wouldn't this therefore not work in r149 where the setting was different?

Reply:
I have a PR for that in #2620, will be cutting a release for these in a moment.